src/applications/vaos/tests/new-appointment/components/ConfirmationPage/index.unit.spec.jsx

```diff
@@ -21,44 +24,87 @@ const startNewAppointmentFlow = sinon.spy();
 
 describe('VAOS <ConfirmationPage>', () => {
   it('should render appointment direct schedule view', async () => {
-    const start = moment().tz('America/Denver');
-    const end = start;
-    const flowType = FLOW_TYPES.DIRECT;
-    const data = {
-      typeOfCareId: '323',
-      phoneNumber: '1234567890',
-      email: 'joeblow@gmail.com',
-      reasonForAppointment: 'routine-follow-up',
-      reasonAdditionalInfo: 'Additional info',
-      vaFacility: '983',
-    };
-    const facilityDetails = {
-      id: 'var983',
-      name: 'Cheyenne VA Medical Center',
-      address: {
-        postalCode: '82001-5356',
-        city: 'Cheyenne',
-        state: 'WY',
-        line: ['2360 East Pershing Boulevard'],
+    const start = moment();
+    const store = createTestStore({
+      newAppointment: {
+        flowType: FLOW_TYPES.DIRECT,
+        data: {
+          typeOfCareId: '323',
+          phoneNumber: '1234567890',
+          email: 'joeblow@gmail.com',
+          reasonForAppointment: 'routine-follow-up',
+          reasonAdditionalInfo: 'Additional info',
+          vaParent: 'var983',
+          vaFacility: 'var983',
+          clinicId: '455',
+          calendarData: {
+            selectedDates: [
+              {
+                datetime: start.format(),
+              },
+            ],
+          },
+        },
+        availableSlots: [
+          {
+            start: start.format(),
+            end: start
+              .clone()
+              .add(30, 'minutes')
+              .format(),
+          },
+        ],
+        parentFacilities: [
+          {
+            id: 'var983',
+            identifier: [
+              { system: 'urn:oid:2.16.840.1.113883.6.233', value: '983' },
+              {
+                system: 'http://med.va.gov/fhir/urn',
+                value: 'urn:va:facility:983',
+              },
+            ],
+          },
+        ],
+        facilityDetails: {
+          var983: {
+            id: 'var983',
+            name: 'Cheyenne VA Medical Center',
+            address: {
+              postalCode: '82001-5356',
+              city: 'Cheyenne',
+              state: 'WY',
+              line: ['2360 East Pershing Boulevard'],
+            },
+          },
+        },
+        clinics: {
+          // eslint-disable-next-line camelcase
+          var983_323: [
+            {
+              id: '455',
+            },
+          ],
+        },
+        facilities: {
+          '323_var983': {
+            id: 'var983',
+            name: 'Cheyenne VA Medical Center',
+          },
+        },
       },
-    };
+    });
 
-    const screen = renderWithStoreAndRouter(
-      <ConfirmationPage
-        facilityDetails={facilityDetails}
-        flowType={flowType}
-        data={data}
-        slot={{ start, end }}
-        systemId="578"
-      />,
-      {
-        initialState,
-      },
-    );
+    const screen = renderWithStoreAndRouter(<ConfirmationPage />, {
```

Comment:
Switched this to render the connected version of ConfirmationPage to make sure we cover the Redux selector logic